src/app/controllers/censusdata/create.js

```diff
@@ -5,7 +5,7 @@ flnd.censusDataCreate = {
 
         $scope.birdSpeciesList = {};
 
-        authService.get(config.birdspecies + "?surveyTypeId=2")        
```

Comment:
@kpunjani at the command line, run `grunt jshint` to see warnings on style errors. In this case, we want to always use  `'` rather than `"`.